src/devices/Card/CreditCard/CreditCard.cs

```diff
@@ -0,0 +1,497 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    /// <summary>
+    /// The Credit Card class
+    /// </summary>
+    public class CreditCard
+    {
+        private const byte Cla = 0x00;
+        private const byte Ins = 0x01;
+        private const byte P1 = 0x02;
+        private const byte P2 = 0x03;
+        private const byte Lc = 0x04;
+
+        private const int MaxBuffer = 260;
+
+        private CardWriteRead _nfc;
+        private bool _alreadyReadSfi = false;
+        private byte _target;
+
+        // This is a string "2PAY.SYS.DDF01" (PPSE) to select the root directory
+        public readonly byte[] RootDirectory = { 0x32, 0x50, 0x41, 0x59, 0x2e, 0x53, 0x59, 0x53, 0x2e, 0x44, 0x44, 0x46, 0x30, 0x31 };
```

Comment:
Why not `Encoding.ASCII.GetBytes`? (or whatever encoding that is)

Reply:
It's just because it's a readonly, it's used only once to select the root directory, it's part of the specifications and always represented like this. I feel like adding ```Encoding.ASCII.GetBytes``` in the constructor won't really add any value at all.